src/compiler/protobuf_plugin.h

```diff
@@ -0,0 +1,261 @@
+
+#ifndef GRPC_INTERNAL_COMPILER_PROTOBUF_PLUGIN_H
+#define GRPC_INTERNAL_COMPILER_PROTOBUF_PLUGIN_H
+
+#include "src/compiler/config.h"
+#include "src/compiler/generator_helpers.h"
+#include "src/compiler/schema_interface.h"
+
+#include <vector>
+
+inline grpc::string DotsToColons(const grpc::string &name) {
```

Comment:
Anything C++ or Python specific should go into cpp_generator.h/python_generator.h.

This applies to all the functions until the next comment.